### ISSUE TITLE
Classification error on imagenet images

### ISSUE THREAD

**Issue by junyongyou** [allanzelener/YAD2K#15]
I am using yolo.cfg and yolo.weights, and then tried some fish images collected from imagenet. However, I found the fishes were both detected (which is excellent) but were classified as birds. Is that because I should use other cfg or weights, or should the model be retrained? Thanks a lot.

**Comment by allanzelener** [comment]
Fish is not one of the 80 COCO classes listed in `model_data/coco_classes.txt` so it's expected that the pretrained model would not be able to classify them even though it was pretrained on Imagenet. 

The simplest thing to do is replace the final layer and finetune the model on a dataset containing fish and whatever other target classes you're interested in, but my code for training is still a work in progress.

**Comment by junyongyou** [comment]
@allanzelener Thanks a lot. I look forward to seeing the updated code.

**Comment by allanzelener** [comment]
I've merged the localization loss branch into master and there is now an example training script called `train_overfit.py`. This can be adapted to write your own training script.

Unfortunately I don't have time to expand this to a full training script for Pascal VOC that fully replicates the Darknet training loop but if you are working on this then that kind of contribution would be welcome.
